cmd/juju/model/exportbundle.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+package model
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+
+	"github.com/juju/juju/api/bundle"
+	"github.com/juju/juju/cmd/modelcmd"
+	"github.com/juju/juju/cmd/output"
+)
+
+// NewExportBundleCommand returns a fully constructed export bundle command.
+func NewExportBundleCommand() cmd.Command {
+	cmd := &exportBundleCommand{}
+	cmd.newAPIFunc = func() (ExportBundleAPI, error) {
+		return cmd.getAPI()
+	}
+	return modelcmd.Wrap(cmd)
+}
+
+type exportBundleCommand struct {
+	modelcmd.ModelCommandBase
+	out        cmd.Output
+	newAPIFunc func() (ExportBundleAPI, error)
+	Filename   string
+}
+
+const exportBundleHelpDoc = `
+Exports the current model configuration as bundle.
+
+If --filename is not used, the configuration is printed to stdout.
+ --filename specifies an output file.
+
+Examples:
+
+    juju export-bundle
+	juju export-bundle --filename mymodel
```

Comment:
Indentation is different between the 2 lines in the Examples section.

Reply:
done